src/core/projectinfo.cpp

```diff
@@ -310,6 +310,8 @@ void ProjectInfo::setActiveLayer( QgsMapLayer *layer )
   mSettings.beginGroup( QStringLiteral( "/qgis/projectInfo/%1" ).arg( mFilePath ) );
   mSettings.setValue( QStringLiteral( "activeLayer" ), layer->id() );
   mSettings.endGroup();
+
+  void activeLayerChanged();
```

Comment:
Is that a new way of calling emit?

Reply:
No, it's a great example of why reviews are most welcome though! :wink: